src/frontend/web_application/src/components/ContactProfile/components/ContactProfileForm/index.jsx

```diff
@@ -73,6 +73,10 @@ class ContactProfileForm extends Component {
 
   render() {
     const { __ } = this.props;
+    const dateProps = {};
+    if (this.state.contact.infos.birthday) {
+      dateProps.selected = new Date(this.state.contact.infos.birthday);
```

Comment:
I guess `this.state.contact.infos.birthday` is already a Date instance, since it's set in the `setState`?